kong/db/dao/keys.lua

```diff
@@ -56,14 +56,24 @@ function keys:each_for_set(foreign_key, size, options)
   return self.super.each_for_set(self, foreign_key, size, options)
 end
 
+local is_empty_field
+do
+  local null    = ngx.null
+  local isempty = require("table.isempty")
+
+  is_empty_field = function(f)
+    return f == nil or f == null or isempty(f)
+  end
+end
```

Comment:
```suggestion
local isempty = require("table.isempty")
```

should be good enough.